src/main/resources/db/migration/R__grant_to_cloudsqliamuser.sql

```diff
@@ -1,4 +1,5 @@
 REVOKE ALL ON ALL TABLES IN SCHEMA public FROM cloudsqliamuser;
 GRANT SELECT ON ALL TABLES IN SCHEMA public TO cloudsqliamuser;
 GRANT SELECT ON ALL TABLES IN SCHEMA public TO "isyfo-analyse";
+-- touch
 -- GRANT SELECT ON ALL TABLES IN SCHEMA public TO "isyfo-analyse";
```

Comment:
Burde disse kommentarene fjernes? 😊

Reply:
Må gjøre en touch her for å få denne til å bli kjørt på nytt - ellers får man ikke tilgang til de nye tabellene som iam-user. Dette er en "quirk" med flyway og repeatable scripts.